src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -1030,43 +1120,35 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
         allocationManager.createOperatorSets(defaultAVS, createSetParams);
         cheats.startPrank(defaultOperator);
         allocationManager.registerForOperatorSets(defaultOperator, registerParams);
+
+        // Allocate
         allocationManager.modifyAllocations(defaultOperator, allocateParams);
         cheats.roll(block.number + DEFAULT_OPERATOR_ALLOCATION_DELAY);
+
+        // Deallocate
         allocationManager.modifyAllocations(defaultOperator, deallocateParams);
         uint32 deallocationEffectBlock = uint32(block.number + DEALLOCATION_DELAY);
         cheats.stopPrank();
 
-        // Check slashable stake after deallocation (still pending; no change)
-        _checkSlashableStake({
-            operatorSet: allocateParams[0].operatorSet,
-            operator: defaultOperator,
-            strategies: allocateParams[0].strategies,
-            expectedStake: allocateParams[0].newMagnitudes[0]
-        });
-
-        // Check slashable stake after deallocation takes effect, before slashing
-        _checkSlashableStake({
-            operatorSet: allocateParams[0].operatorSet,
-            operator: defaultOperator,
-            strategies: allocateParams[0].strategies,
-            expectedStake: deallocateParams[0].newMagnitudes[0],
-            futureBlock: deallocationEffectBlock
-        });
-
         uint256 magnitudeAllocated = allocateParams[0].newMagnitudes[0];
         uint256 magnitudeDeallocated = magnitudeAllocated - deallocateParams[0].newMagnitudes[0];
-        uint256 magnitudeSlashed = magnitudeAllocated.mulWad(slashingParams.wadsToSlash[0]);
+        uint256 magnitudeSlashed = (magnitudeAllocated * slashingParams.wadsToSlash[0] / WAD) + _calculateSlippage(uint64(magnitudeAllocated), slashingParams.wadsToSlash[0]);
         uint256 expectedCurrentMagnitude = magnitudeAllocated - magnitudeSlashed;
         int128 expectedPendingDiff =
             -int128(uint128(magnitudeDeallocated - magnitudeDeallocated.mulWadRoundUp(slashingParams.wadsToSlash[0])));
 
-        _checkSlashEvents({
-            operator: defaultOperator,
-            operatorSet: allocateParams[0].operatorSet,
-            strategies: allocateParams[0].strategies,
-            wadsToSlash: slashingParams.wadsToSlash,
-            description: "test"
-        });
+        // Manually check slash events since we have a deallocation pending
+        // Deallocation update is emitted first
+        cheats.expectEmit(true, true, true, true, address(allocationManager));
```

Comment:
Possible to replace with `_checkSlashEvents`?

Reply:
Unfortunately we cannot since a deallocation also emits an additional event interspersed with the other events. Would need a helper but are only testing this twice so figure its not needed rn